### ISSUE TITLE
Logging card numbers!

### ISSUE THREAD

**Issue by aehven** [paypal/PayPal-Ruby-SDK#146]
Line 92 of http_helper.rb recently introduced a debug log that causes the entire payload, including sensitive info to be logged.  It is conceivable that an application in production will be running for a while with debug logging turned on.  This line should be removed.

**Comment by juwlee** [comment]
@aehven thanks for feedback. We introduced that line in order to help developers easily capture the request and the response for issue investigation. For that reason, I think it might be better not to log payload by default, but when `verbose_logging` is enabled. (See commit fd08622 for all related changes.) It should be pretty safe with two switches that enable payload print which are set to false by default.

**Comment by aehven** [comment]
@juwlee Just wanted to see thanks for your quick response as soon as possible.  Will test this later today or tomorrow.

**Comment by aehven** [comment]
@juwlee Tested and it works great, thanks again.  

I was unable to get the 1.3.3 branch by specifying a branch in my Gemfile.  I had to specify the ref of your last commit.  Don't know why that is.  Any idea when this will be out as a stable release or merged into master?

Thanks again,

-- Adam

**Comment by juwlee** [comment]
@aehven We are adding a few more enhancements and fixes at the moment and will release a new version with all of them included. It should not be too long.

**Comment by aehven** [comment]
Hi @juwlee,

Looking at the fix you've put in 1.3.3, it looks like it all depends on forcing debug level to INFO for live mode in the logger= method.  But for some reason, in my system when I'm live, this function is never called.  Maybe because I already have a logger defined.  Am I doing something wrong?  Is this fix not useful if you already have a logger defined?

Thanks,

-- Adam

**Comment by aehven** [comment]
Interestingly, it seems that I can get around this by having an initializer with
````
PayPal::SDK.logger = Rails.logger
PayPal::SDK.logger.level = Logger::INFO
````
but then that also requires the existence of a paypal.yml which I wouldn't generally need because all my config options have to be figured out during run-time.  Does this make sense?
